sdk/workflow_run.go

```diff
@@ -22,6 +22,8 @@ type WorkflowRun struct {
 	WorkflowNodeRuns map[int64][]WorkflowNodeRun `json:"nodes" db:"-"`
 	Infos            []WorkflowRunInfo           `json:"infos" db:"-"`
 	Tags             []WorkflowRunTag            `json:"tags" db:"-"`
+	LastSubNumber    int64                       `json:"last_subnumber" db:"last_sub_num"`
```

Comment:
why not `last_sub_num` as for db ? Or `last_sub_number` but change `num` to number for version number.